internal/plugins/helm/v1/init.go

```diff
@@ -199,7 +199,7 @@ func addInitCustomizations(projectName string) error {
 	}
 
 	// Increase the default memory required.
-	err = sdkutil.ReplaceInFile(managerFile, "memory: 30Mi", "memory: 90Mi")
+	err = sdkutil.ReplaceInFile(managerFile, "memory: 100Mi", "memory: 90Mi")
```

Comment:
I would just get rid of this replacement since the new limits are desirable.

Reply:
Does this need to be done for ansible too?

Reply:
Nope, ansible does not make the change to manager's resource limits.